charts/jenkins/Chart.yaml

```diff
@@ -33,6 +33,6 @@ annotations:
     - name: k8s-sidecar
       image: kiwigrid/k8s-sidecar:1.15.0
     - name: inbound-agent
-      image: jenkins/inbound-agent:4.11.2-4
+      image: jenkins/inbound-agent:latest
```

Comment:
why latest? wouldn't it be be better to fix the version and then have renovate / update cli update it?

Reply:
Right, good catch!